pkg/cmd/pulumi/pulumi.go

```diff
@@ -585,11 +585,14 @@ func getCLIVersionInfo(
 	metadata map[string]string,
 ) (semver.Version, semver.Version, semver.Version, int, error) {
 	creds, err := workspace.GetStoredCredentials()
+	apiToken := creds.AccessTokens[creds.Current]
```

Comment:
I think this map lookup could potentially fail

Reply:
It could but it will just return empty in that case.